integration/testnet/network.go

```diff
@@ -100,8 +101,6 @@ const (
 	MetricsPort = "8080"
 	// AdminPort is the admin server port
 	AdminPort = "9002"
-	// ExecutionStatePort is the execution state server port
-	ExecutionStatePort = "9003"
```

Comment:
The SDK has support for the `executiondata` API now, but requires that it's served on the same port as the access API. I opted to just update all of the tests to use the same port for simplicity